Lib/importlib/__init__.py

```diff
@@ -36,3 +36,5 @@ def import_module(name, package=None):
         name = _resolve_name(name[level:], package, level)
     __import__(name)
     return sys.modules[name]
+
+reload = reload
```

Comment:
Why do you need this?

Reply:
This adds `importlib.reload`, by default in Python 2 `reload` is a builtin.